plugins/ConditionalRestrictions.py

```diff
@@ -42,6 +42,8 @@ def init(self, logger):
     # Workaround https://bugs.kde.org/show_bug.cgi?id=452236
     self.kOpeningHours452236 = re.compile(r'(20\d\d [A-Z][a-z][a-z] \d\d?\s?-\s?)20\d\d ([A-Z][a-z][a-z] \d\d?)')
 
+    self.getDuplicatesFromList = lambda list: set([x for x in list if list.count(x) > 1])
```

Comment:
Better make the set before. Something like
```suggestion
    self.getDuplicatesFromList = lambda list: [x for x in set(list) if list.count(x) > 1]
```

Reply:
Done; also renamed list to lst to not unnecessarily overwrite any python function